src/lib.rs

```diff
@@ -125,9 +125,10 @@ impl<T> RofiChild<T> {
 
 impl RofiChild<String> {
     /// Wait for the result and return the output as a String.
-    fn wait_with_output(&mut self) -> Result<String, Error> {
+    fn wait_with_output(&mut self) -> Result<(i32, String), Error> {
         let status = self.p.wait()?;
-        if status.success() {
+        let code = status.code().unwrap();
+        if status.success() || (code >= 10 && code <= 30)  {
```

Comment:
Could you explain why you check the exit code to be between 10 and 30?

Reply:
Sure! When you set and use a custom keyboard shortcut in rofi the return code will correspond to the used shortcut. This information will not be provided in the stdout.

The limits I've set here are not exactly correct has you can see here under ROFI_RETV: https://davatorium.github.io/rofi/1.7.5/rofi-script.5/

I'll clean this up and ping you soon